content/docs/intro/concepts/stack.md

```diff
@@ -4,14 +4,16 @@ meta_desc: An in depth look at Pulumi Stacks and their usage.
 menu:
   intro:
     parent: concepts
-    weight: 3
+    weight: 2
 
 aliases: ["/docs/reference/stack/"]
 ---
 
-Every Pulumi program is deployed to a **stack**.  A stack is an isolated, independently [configurable]({{< relref "config" >}})
+Every Pulumi program is deployed to a **stack**. A stack is an isolated, independently [configurable]({{< relref "/docs/intro/concepts/config" >}})
```

Comment:
Nit: we rarely use bold in text blocks in general, and for new terms like this, italics seems typical. Suggest we stick with that if possible.

```suggestion
Every Pulumi program is deployed to a _stack_. A stack is an isolated, independently [configurable]({{< relref "/docs/intro/concepts/config" >}})
```